hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseTokenHandler.java

```diff
@@ -262,25 +263,25 @@ private List<TokenRelation> createTokenRelsToAccount(
             // Compute the previous and next token IDs. Unfortunately `TokenRelation` doesn't
             // allow for null values, so a value of '0' will have to indicate a null pointer to
             // the previous or next token (since no token number 0 can exist)
-            long prevTokenId = 0;
-            long nextTokenId = 0;
+            TokenID prevTokenId = TokenID.DEFAULT;
+            TokenID nextTokenId = TokenID.DEFAULT;
```

Comment:
We should make both these null
```suggestion
            TokenID prevTokenId = null;
            TokenID nextTokenId = null;
```

Reply:
Updated, thanks.